TalkAloud/AudioSlider.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  AudioSlider.swift
+//  TalkAloud
+//
+//  Created by Justin Bengtson on 5/12/20.
+//  Copyright © 2020 Justin Bengtson. All rights reserved.
+//
+
+import UIKit
+
+protocol AudioSliderDelegate: class {
+    func didChangeScrolling(in audioSlider: UISlider)
+    func didBeginScrolling(in audioSlider: UISlider)
+    func didEndScrolling(in audioSlider: UISlider)
+}
+
+class AudioSlider: UISlider {
+    
+    weak var delegate: AudioSliderDelegate?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        addTargets()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        addTargets()
+    }
+    
+    private func addTargets() {
+        addTarget(self, action: #selector(didStartScrubbing(_:)), for: .editingDidBegin)
+        addTarget(self, action: #selector(sliderValueChanged(_:)), for: .valueChanged)
+        addTarget(self, action: #selector(didEndScrubbing(_:)), for: .valueChanged)
```

Comment:
You are using .valueChanged for both sliderValueChanged and didEndScrubbing.

Reply:
Yeah, having didEndScrubbing wouldn't let my audioTime change based on the slider value at this point, I'll look to see what I can do though.

Reply:
According to the docs the methods `editingDidBegin` and `editingDidEnd` are only applied when changes are being made in a UITextField.

Reply:
Need to find a better solution then if you want to have proper scrubbing :)